test/helpers/operating_systems.py

```diff
@@ -89,3 +89,65 @@ def verifyDualBootDebian(self, root_one_size=None, root_two_size=None):
             self.assertEqual(vda15["mountpoints"], ["/boot/efi"])
         else:
             self.assertEqual(vda15["mountpoints"], ["/efi", "/boot/efi"])
+
+
+class DualBootHelperFedora_E2E(VirtInstallMachineCase):
```

Comment:
E2E can be removed. It's just naming convention for test classes.

Also you can adjust a few lines above DualBootHelper to be DualBootHelperDebian, and make sure there is not some methods you could pottentially reuse.